### PR TITLE
Added Popup window

### PR DESCRIPTION
**Pull request by naanmohammed**
In this milestone the following functionalities were added :

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- The popup window was implemented.
- Popups for both mobile and desktop screens were implemented

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,10 +135,66 @@ <h1>Multi-Post <br> Stories</h1>
           <li class="ruby_tag">Ruby on rails</li>
           <li>javaScript</li>
         </ul>
-        <button>See Project</button>
+        <button onclick="showPopup(3)">See Project</button>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure that the projects section of the page is generated dynamically using Javascript. Please remove all HTML content on the project section and generate with Javascript.

File: index.html [hunk 2]
```diff
@@ -135,10 +135,66 @@ <h1>Multi-Post <br> Stories</h1>
...
+            <li id="popup-tag1"></li>
+            <li id="popup-tag2"></li>
+            <li id="popup-tag3"></li>
+          </ul>
+          <ul class="popup-technologies_2">
+            <li id="popup-tag4"></li>
+            <li id="popup-tag5"></li>
+          </ul>
+          <hr/>
+          <button class="popup-btn" id="popup-btn1" onclick="liveLink()">See live <img src="images/live.png"></button>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure that the popup of each project is generated dynamically using Javascript when a user clicks on the `See project` button. Please remove all HTML content on the popup and generate with Javascript.

File: index.html [hunk 3]
```diff
@@ -66,7 +66,7 @@ <h1>Tonic</h1>
           <li>css</li>
           <li>javaScript</li>
         </ul>
-        <button>See Project</button>
+        <button onclick="showPopup(0)">See Project</button>
```
Comment (AnselemOdims) [inline index.html]: - [ ] At the moment, when you click on the `See project` button nothing happens. Please ensure that all works fine before requesting for another review.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @naanmohammed , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css [hunk 4]
```diff
@@ -489,6 +489,104 @@ label {
...
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 75%;
+  height: 90%;
+  background-color: white;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+}
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] You have done a great job. Allow me to say congratulations 🎆 👏 . The popup window is working perfectly as expected. However, we need to work on the slight design to make it more user-friendly. I have noticed the modal content crosses the borders at around `868px` and `590px` screen sizes. Kindly consider changing the height to ensure that all the contents remain with in the card at all screen sizes ☺️ 👍.

Your design 😎 
image

image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @naanmohammed 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comment.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Comment by naanmohammed** [comment]
As per the suggestions of @AlphaNtihinduka I have fixed the overflow using overflow : scroll;

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @naanmohammed,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**